tests/galsim_test_helpers.py

```diff
@@ -361,7 +361,8 @@ def all_obj_diff(objs):
         for j, objj in enumerate(objs):
             if i == j:
                 continue
-            assert obji != objj, "Found equivalent objects {} == {}".format(obji, objj)
+            assert obji != objj, ("Found equivalent objects {} == {} at indices {} and {}"
+                                  .format(obji, objj, i, j))
```

Comment:
Of course, it's a lot easier to upgrade scipy than python.  `pip install scipy --upgrade --prefix=~` is pretty straightforward.  Installing a new python is quite a bit more involved.  Nonetheless, I think I probably agree with you.